### ISSUE TITLE
Conjunctivochalasis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10876]
Preferred term label:
Conjunctivochalasis

Synonyms
Conjunctival folding, Horizontal conjunctival folds, Conjunctival laxity

Definition (free text, please give PubMed ID)
A condition characterized by redundant, loose conjunctival tissue, typically located between the globe and the lower eyelid, which may cause ocular discomfort, tear film instability, or epiphora.

Parent term (use hpo.jax.org/app)
Abnormality of the conjunctiva (HP:502)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Dry eye, Floppy eyelid syndrome

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001233
- label: Conjunctivochalasis
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: 
- def: A condition characterized by redundant, loose conjunctival tissue, typically located between the globe and the lower eyelid, which may cause ocular discomfort, tear film instability, or epiphora.
- comment: Conjunctivochalasis can be observed with conditions such as dry eye and floppy eyelid syndrome.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10876
- synonyms:
